integration/bbi_sge_test.go

```diff
@@ -14,117 +14,121 @@ import (
 )
 
 func TestBbiCompletesSGEExecution(tt *testing.T) {
-	t := wrapt.WrapT(tt)
-
-	// Get BB and make sure we have the test data moved over.
-	// Clean Slate
-
 	if !FeatureEnabled("SGE") {
-		t.Skip("Skipping SGE as it's not enabled")
+		tt.Skip("Skipping SGE as it's not enabled")
```

Comment:
is there a reason this isn't using the `SkipIfNotEnabled` helper from the previous tests?

Reply:
Probably not, I didn't want to get into the inconsistencies in the tests yet because a lot of them could be merged together into a bigger more flexible table test.